logfire/_internal/integrations/httpx.py

```diff
@@ -338,3 +359,34 @@ def capture_request_body(span: Span, request: RequestInfo) -> None:
     attr_name = 'http.request.body.json'
     set_user_attributes_on_raw_span(span, {attr_name: {}})  # type: ignore
     span.set_attribute(attr_name, body)
+
+
+CODES_FOR_METHODS_WITH_DATA_PARAM = [
+    inspect.unwrap(method).__code__
+    for method in [
+        httpx.Client.request,
+        httpx.Client.stream,
+        httpx.AsyncClient.request,
+        httpx.AsyncClient.stream,
+    ]
+]
+
+
+def capture_request_form_data(span: Span, request: RequestInfo) -> None:
+    content_type = cast('httpx.Headers', request.headers).get('content-type', '')
+    if content_type != 'application/x-www-form-urlencoded':
```

Comment:
and `multipart/form-data`?

Reply:
Based on the httpx source code (encode_request -> encode_urlencoded_data) it's always this, multipart is for `files`.